bogota-abril-2019/css-magazine-cover/laura_bruges-task2-magazine/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>NatGeo black and white</title>
+    <link href="https://fonts.googleapis.com/css?family=Old+Standard+TT:400,700|Work+Sans:500" rel="stylesheet"> 
+    <link rel="stylesheet" type="text/css" href="styles/magazine.css"/>
+</head>
+<body>
+    <article id="margin">
+        <aside id="girls-container" class="container"></aside>

```

Comment:
Do you really need this empty element?

Reply:
That element is for the background overlapping (the girls heads over the natgeo title). Please refer to the css.